v3-core/contracts/libraries/Position.sol

```diff
@@ -27,15 +27,53 @@ library Position {
     /// @param tickLower The lower tick boundary of the position
     /// @param tickUpper The upper tick boundary of the position
     /// @return position The position info struct of the given owners' position
+
+    /// warp-cairo
+    /// func INTERNALFUNC(get){
+    ///     syscall_ptr : felt*,
+    ///     pedersen_ptr : HashBuiltin*,
+    ///     range_check_ptr : felt,
+    ///     bitwise_ptr : BitwiseBuiltin*,
+    ///     warp_memory : DictAccess*,
+    ///     keccak_ptr : felt*,
+    /// }(
+    ///     __warp_usrid6_self : felt,
+    ///     __warp_usrid7_owner : felt,
+    ///     __warp_usrid8_tickLower : felt,
+    ///     __warp_usrid9_tickUpper : felt,
+    /// ) -> (__warp_usrid10_position : felt):
+    ///     from warplib.memory import wm_alloc, wm_write_felt
+    ///     alloc_locals
+    ///     let __warp_usrid10_position = 0
+    ///     let (__warp_se_0) = wm_alloc(Uint256(299, 0))
```

Comment:
I think we would rather use wm_new since it creates a dynarray in memory. Then you can wm_write_felt the three values into the 3 new felt slots and then pass the initial location to warp_keccak.